leader-election/Readme.md

```diff
@@ -4,36 +4,79 @@ This document describes the Leader Election Pattern example from the guide [Clou
+
+### Running with Azure Storage account
+
+Using the Azure portal, create a new Azure Storage account, or us an existing account.
+
+The sample defaults to use a container named `leases`. If you are utilizing an existing storage account, be certain that this container is not already in use.
+
+Find the connection string in your Storage Account's `Access keys` section in the portal. Copy the Connection string value from either key1 or key2 and replace the `StorageConnectionString` value field in `LeaderElectionConsoleWorker/app.config` with the copied value.
+
+### Running the worker process
+
+Navigate to the `leader-election/LeaderElectionConsoleApp` directory, and start the application
+
+```shell
+cd LeaderElectionConsoleWorker
+dotnet run
+```
+
+You can start multiple instances of the worker process by opening additional terminal windows or tabs, and running the same command. When a worker process is the leader, you will see periodic output like:
+
+```output
+[14:22:30] This process (51635) is currently the leader. Press any key to exit.
+```
+
+If a worker process is not the leader, you will see:
 
+```output
+[14:23:21] This process (51686) could not acquire lease. Retrying in 20 seconds. Press any key to exit.
+```
 
-* Start Visual Studio using an account that has Administrator privileges ("Run as Administrator").
-* Open the solution you want to explore from the subfolders where you downloaded the examples.
-* Right-click on each role in Solution Explorer, select Properties, and ensure that the role is configured to generate diagnostic information.
+You can terminate the current leader, and watch the other processes to see one of them acquire the lease and become the new leader:
 
-* If you want to run the example in the local Windows Azure emulator:
-	* Press F5 in Visual Studio to start the example running.
-	* Open the Windows Azure Compute Emulator UI from the icon in the notification area.
-	* Select each role in turn and view the diagnostic information generated by Trace statements in the code.
-* If you want to run the example on Windows Azure:
-	* Provision a Windows Azure Cloud Service and deploy the application to it from Visual Studio.
-	* Open the Server Explorer Window in Visual Studio and expand the Windows Azure entry.
-	* Expand Cloud Services and then expand the entry for the solution you deployed.
-	* Right-click each role instance and select View Diagnostics Data to see the diagnostic information generated by Trace statements in the code. This is written to the WADLogsTable in the Storage/Development/Tables section.
+```output
+[14:24:21] This process (51686) could not acquire lease. Retrying in 20 seconds. Press any key to exit.
+[14:24:41] This process (51686) could not acquire lease. Retrying in 20 seconds. Press any key to exit.
+[14:25:01] This process (51686) is currently the leader. Press any key to exit.
+```
```

Comment:
Can you tweak this to a bit more closely follow the readme pattern presented on https://github.com/mspnp/cloud-design-patterns/tree/main/pipes-and-filters

Intro (including summary of what it does)
Deployment guide
Prerequisites (bullets)
Steps (numbered)
Try it out (numbered)
Clean up resources (if they selected to use the Azure storage account path)
Related documentation
Contributions

Reply:
Looks great!